### DISCUSSION TITLE
disable `changesets` bot/application

### DISCUSSION THREAD

**Discussion by uiolee** [hexojs/hexo#5404] [General]
https://github.com/hexojs/hexo-uglify/pull/404#issuecomment-1902936536
https://github.com/hexojs/site/pull/2135#issuecomment-1902720772

It looks like someone installed [`changesets`](https://github.com/apps/changeset-bot) bot/application for all repos, but this is not necessary.

Only [hexojs/hexo-monorepo](https://github.com/hexojs/hexo-monorepo) requires it for testing.

Please disable it, thanks you. ❤️

**Comment by yoshinorin** [comment]
@uiolee 
Done :)

![changeset](https://github.com/hexojs/hexo/assets/11273093/aa2ae7e5-a801-44a2-898d-5cc55c7605f0)

**Reply by uiolee** [reply]
thank you :)
